docs/source/newsfragments/3668.feature.rst

```diff
@@ -1 +1 @@
-Add `log_file` argument to :ref:`Python Test Runner <howto-python-runner>` to redirect stdout and stderr to the specified file.
+The :meth:`cocotb_tools.runner.Simulator.build` and :meth:`cocotb_tools.runner.Simulator.test` methods now accept a ``log_file`` argument to redirect stdout and stderr to the specified file.
```

Comment:
Do we want to use this fully qualified syntax or use ``:meth:`Simulator.build() <cocotb_tools.runner.Simulator.build>` ``?

I think we should be consistent.

Reply:
I left `runner` in there because the base class is called `Simulator` but we refer to this as the Python runner. Maybe we should add a `Runner` suffix to these class names.

Reply:
I'd be okay with straight up renaming `Simulator` to `Runner`.